docker-compose.yml

```diff
@@ -121,6 +121,7 @@ services:
             - LETSENCRYPT_DOMAIN
             - LETSENCRYPT_EMAIL
             - LETSENCRYPT_USE_STAGING
+            - LETSENCRYPT_ACME_DIRECTORY
```

Comment:
I think this variable name is very misleading. acme.sh may implement the feature using a directory, but that's not what the user really wants, they want to change the server. Let's call this LETSENCRYPT_ACME_SERVER.